### DISCUSSION TITLE
how to register class var in uno

### DISCUSSION THREAD

**Discussion by manyuemeiquqi** [unocss/unocss#3761] [Q&A] [answered]
```Vue
<script setup lang="ts">
import { ref } from "vue";

defineProps<{ msg: string }>();

const count = ref(0);

let classStr = "h-fu";
classStr += "ll";
</script>

<template>
  <div :class="classStr">121</div>
</template>

<style scoped>
.read-the-docs {
  color: #888;
}
</style>

```

If i write this in file, h-full doesn't work in vite, and how to resolve this problem?

**Comment by schelmo** [comment, marked as answer]
in the docs there is the explanation why your example is not working:
https://unocss.dev/guide/extracting#safelist
> Due the fact that UnoCSS works in build time using static extraction, at the compile time it can't possibility know all the combination of the utilities then. For that, you can configure the safelist option.
